company-profiles/memory.md

```diff
@@ -0,0 +1,30 @@
+# Memory
+
+## Company blurb
+
+Memory is not your average Norwegian company. We work tirelessly because we have a need build something bigger than ourselves. To create.
```

Comment:
Typo: "because we have a need **to** build"

Reply:
Fixed in https://github.com/remoteintech/remote-jobs/pull/618/commits/cc608f585280e66454b2039a1555db4e8cbe4df0